<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgBox</class>
 <widget class="QWidget" name="MsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MsgBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_main" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_top" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_text">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnYes">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNo">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
